### PR TITLE
Update Go docs

### PR DESCRIPTION
**Pull request by tangowithfoxtrot**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/SM-1265

## 📔 Objective

Correct a syntax error and use consistent and descriptive `bitwardenClient` variable name.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: languages/go/README.md [hunk 1]
```diff
@@ -52,25 +52,25 @@ err := bitwardenClient.AccessTokenLogin(accessToken, &statePath)
 #### Create a Project
 
 ```go
-project, err := client.Projects().Create("organization_id", "project_name")
+project, err := bitwardenClient.Projects().Create("organization_id", "project_name")
 ```
 
 #### List Projects
```
Comment (mzieniukbw) [inline languages/go/README.md:58]: ⛏️ Should we add Get Project (by id) ?

File: languages/go/README.md [hunk 2]
```diff
@@ -80,34 +80,34 @@ project, err := client.Projects().Delete([]string{"project_id_1", "project_id_2"
 #### Create a Secret
 
 ```go
-secret, err := client.Secrets().Create("key", "value", "note", "organization_id", []string{"project_id"})
+secret, err := bitwardenClient.Secrets().Create("key", "value", "note", "organization_id", []string{"project_id"})
 ```
 
 #### List Secrets
```
Comment (mzieniukbw) [inline languages/go/README.md:86]: ⛏️ Should we add Get Secrets (by id) and Get (multiple) Secrets By Ids ?

**Review by mzieniukbw — DISMISSED** [review]
LGTM.

File: languages/go/README.md (hunk 1 above)
Comment (tangowithfoxtrot) [reply languages/go/README.md:58]: Great catch! Added in 2ec353f.

File: languages/go/README.md (hunk 2 above)
Comment (tangowithfoxtrot) [reply languages/go/README.md:92]: Added in 47f059e.

**Review by mzieniukbw — APPROVED** [review]

**Review by coltonhurst — APPROVED** [review]

**Pull request merged**